lsp-ui-doc.el

```diff
@@ -702,12 +702,18 @@ FN is the function to call on click."
 (defvar-local lsp-ui-doc--inline-width nil)
 
 (defun lsp-ui-doc--inline-window-width nil
+  "Calculate inner padding for the child frame showing the documentation"
+  (if (not (bound-and-true-p lsp-ui-doc-use-childframe))
+      (setq additional-width-spacing 8)
+    (setq additional-width-spacing 1))
+
   (- (min (window-text-width)
           (window-body-width))
      (if (bound-and-true-p display-line-numbers-mode)
          (+ 2 (line-number-display-width))
-       0)
-     1))
+         0)
+     additional-width-spacing)
+  )
```

Comment:
Can you use `let` operator for local variable `additional-width-spacing`? And don't put `)` on a newline (L716). Thanks!

Reply:
done